### PR TITLE
net-misc/eventd: verbump to 0.23.0

### PR DESCRIPTION
**Pull request by Coacher**
Upstream now uses meson build system exclusively, so port ebuild to it.
Obsoletes PR #4524.

### CODE REVIEW

File: net-misc/eventd/eventd-0.23.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	app-text/docbook-xsl-stylesheets
+	dev-libs/libxslt
+	virtual/pkgconfig
+	fbcon? ( virtual/os-headers )
+"
+RDEPEND="${COMMON_DEPEND}
+	net-libs/glib-networking[ssl]
+"
+
+eventd_check_compiler() {
```
Comment (mgorny) [inline net-misc/eventd/eventd-0.23.0.ebuild]: Those compiler versions are p.masked now (yeah!), so drop the check ;-).

File: net-misc/eventd/eventd-0.23.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic linux-info meson ninja-utils systemd xdg-utils
```
Comment (mgorny) [inline net-misc/eventd/eventd-0.23.0.ebuild]: I don't see you calling eninja, so ninja-utils are not a direct dep.

Comment (Coacher) [inline net-misc/eventd/eventd-0.23.0.ebuild]: Ah, yes. Leftovers from when meson.eclass wasn't in tree.

File: net-misc/eventd/eventd-0.23.0.ebuild (hunk 1 above)
Comment (Coacher) [inline net-misc/eventd/eventd-0.23.0.ebuild]: Nah. I'd rather keep it until those versions are gone completely.

File: net-misc/eventd/eventd-0.23.0.ebuild (hunk 2 above)
Comment (Coacher) [inline net-misc/eventd/eventd-0.23.0.ebuild]: Done.

File: net-misc/eventd/eventd-0.23.0.ebuild (hunk 1 above)
Comment (mgorny) [inline net-misc/eventd/eventd-0.23.0.ebuild]: I will let you leave them for this time but next time, please really do get rid of this. The purpose of this mask was to get rid of all the checks. `pkg_pretend()` is slow, and doing meaningless checks for unsupported systems is a waste of everyone's time.

**Comment by Coacher** [comment]
Thank you, @mgorny.

**Pull request closed without merging**